widgetry/src/widgets/fan_chart.rs

```diff
@@ -133,7 +133,7 @@ impl FanChart {
             p99.reverse();
             band.extend(transform(p99));
             band.push(band[0]);
-            batch.push(s.color.alpha(0.5), Polygon::buggy_new(band));
+            batch.push(s.color.alpha(0.5), Tessellation::triangulate(band));
```

Comment:
![Screenshot from 2022-08-16 21-17-34](https://user-images.githubusercontent.com/1664407/184979094-50f79ea3-a3fa-4fdc-84b8-9a6ade7f5e4c.png)
I had almost forgotten what this thing does!

Reply:
Is `band` a Ring of Coordinates? If so maybe a better name would be:

```diff
- pub fn triangulate(points: Vec<Pt2D>) -> Self {
+ pub fn from_ring(points: Vec<Pt2D>) -> Self {
```

It's already kind of implied that it's going to be triangulated since it's a Tessellation and all.

Reply:
Done. I'll add a docstring explaining that it's not a `Ring` because there may be duplicate points or other issues, but it's likely still triangulatable